src/main/webapp/WEB-INF/web.xml

```diff
@@ -3,7 +3,7 @@
 	xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd"
 	version="2.4">
 	<display-name>CanvasCourseManagerl</display-name>
```

Comment:
Why is the display name named as `CanvasCourseManagerl` why is the extral one or l is added?